src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.Tests/assets/SdkContentTests/SdkFileDiffExclusions.txt

```diff
@@ -83,6 +83,10 @@
 ./sdk/x.y.z/*/Microsoft.VisualStudio.TestPlatform.Extensions.Html.TestLogger.resources.dll|sb
 ./sdk/x.y.z/*/Microsoft.VisualStudio.TestPlatform.Extensions.Trx.TestLogger.resources.dll|sb
 
+# https://github.com/dotnet/source-build/issues/4881
```

Comment:
The new tracking issue is good. But I think it should be the new tracking issue for all the existing exclusions above that are tracked by https://github.com/dotnet/source-build/issues/4079. So I think you just need to replace line 80 with the new tracking issue and add the new exclusions under that header.